packages/cli/src/Version.ts

```diff
@@ -90,17 +93,38 @@ export class Version implements Command {
     })
 
     const prismaClientVersion = await getInstalledPrismaClientVersion()
-
-    const rows = [
+    const schemaPath = await getSchemaPath()
```

Comment:
Note, `getSchemaPath`'s return type changed recently.

```suggestion
    const { schemaPath } = await getSchemaPath()
```